authentication/using-service-accounts-in-applications.adoc

```diff
@@ -10,7 +10,7 @@ include::modules/service-accounts-overview.adoc[leveloffset=+1]
 
 include::modules/service-accounts-default.adoc[leveloffset=+1]
 
-// remove these links for 4.12+
```

Comment:
What was this?

Reply:
The writer who added these docs marked that they should probably be removed in 4.12 (because they were mainly talking about the change in behavior between 4.10 and 4.11).

But since I've changed it to talk about the current behavior (and only add a note of the change in behavior), I think this is fine to keep in here through all current/future releases.